protocol/protobuf/pairing.proto

```diff
@@ -420,6 +420,7 @@ message SyncKeycard {
 message SyncSocialLinks {
   repeated SocialLink social_links = 1;
   uint64 clock = 2;
+  option deprecated = true;
```

Comment:
Why do we need that? :thinking:

Reply:
Deprecation for whole message, based on this stack overflow answer: https://stackoverflow.com/questions/52781727/how-can-i-deprecate-whole-message-in-protocol-buffers